test/unittests/test_async_client.py

```
@@ -0,0 +1,454 @@
+            mock_ws.connect = AsyncMock(return_value=ws_mock)
+            mock_ws.exceptions = MagicMock()
+            await bus.connect(retry=False)
+
+        self.assertEqual(len(open_events), 1)
+        # cleanup
+        if bus._listen_task:
+            bus._listen_task.cancel()
+            try:
+                await bus._listen_task
+            except (asyncio.CancelledError, Exception):
+                pass
```

Ruff (0.15.5)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)